src/scss/components/_expandable-content.scss

```diff
@@ -0,0 +1,15 @@
+.c-expandable-content {
```

Comment:
This stuff is included in the system, but it isn't documented. This feels like more of a _concept_ than a _component_, so I think this would be best to explain with a page under "Documentation". Thoughts?

Reply:
Why wouldn't we make a component in the markup section for it? The JS that goes with the toggler is also included we might as well document it.

Reply:
I totally thought I had a reason why it didn't fit well as a component... but I can't think of it today! Maybe I was feeling lazy and didn't want to think up a collection for it to belong in? 🤷‍♂ 

I'll add it!

Reply:
As I'm adding this, I'm re-remembering why I felt conflicted about adding it in the first place. It's hard to communicate which classes, attributes, and elements are required, and which ones are added just to make it look a little nicer.

This is important to show, because devs building on top of this should be empowered to add styles to make this match whatever brand they're building for.

I'll try to handle explaining this within the pattern's `notes`. In the future it'd be nice to break the pattern into "simplest possible version" to show what's _required_. And _then_ show a slightly beefed-up version that shows the `&--flip-when-expanded` modifier, content with more styles applied, etc.